docs/src/pages/company/software-engineer-intern/software-engineer-intern.md

```diff
@@ -0,0 +1,78 @@
+# Software Engineer Intern
+
+<p class="description">We are looking for software engineer intern to help support our open-source components, assist the Material-UI community, and help us build new components.</p>
+
+## Details of the Role
+
+- Location: Remote (preference for UTC-5 to UTC+4).
+- Type of work: Full-time internship.
+- Start date: Flexible.
+- We're a remote company, we prefer asynchronous communication over meetings.
+- We work independently, the rest of us won't know what you're doing day-to-day unless you tell us.
+
+## About Us
+
+See the [careers](/company/careers/) and [about us](https://next.material-ui.com/branding/about/) pages.
+
+### Why we're hiring
+
+As our teams are growing (x2-3 YoY), we thought that it's time to give chance for new talented engineers to join us for an internship!
```

Comment:
I would propose we frame the messaging more toward there are senior members that can mentor, coach you.

```suggestion
We have recently grown the engineering teams with senior members. It's time to give chance for a new talented junior engineer to join us for an internship!
```